<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>460</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Service Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowserLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>641</width>
      <height>190</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;
color:white;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Terminal'; font-size:6pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>645</width>
      <height>245</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_flash">
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>209</y>
        <width>640</width>
        <height>28</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelVersion">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>2</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="btnFwDownload">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Firmware Upload</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="btnCertDownload">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Certificate Upload</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnWrite">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConnect">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelBuild">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>35</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxProject">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>115</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>TCU 2G</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxComPort">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>484</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Select Serial Port...</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnCfgRecover">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>115</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Config Recovery</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_mid">
     <widget class="QLabel" name="labelWait">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Please Wait. Loading...</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLoading">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_cfg">
     <widget class="QPushButton" name="btnGetVariant">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>77</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GET Variant</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelVariant">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>73</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Variant</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNwsw">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>152</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NW Switching</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNwsw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>153</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetnSetNwsw">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>151</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Verify NWSW</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSimtype">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>181</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SIMTYP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSimtype">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>182</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetnSetSimtype">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>181</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Verify SIMTYP</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCip2">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>70</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CIP2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCip2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>240</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetCip2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Verify CIP2</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUin">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>100</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>UIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>99</y>
        <width>166</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetUin">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>124</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GET UIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetUin">
      <property name="geometry">
       <rect>
        <x>462</x>
        <y>124</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SET UIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVin">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>150</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditVin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>166</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetVin">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>175</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GET VIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetVin">
      <property name="geometry">
       <rect>
        <x>462</x>
        <y>175</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SET VIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReboot">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>212</y>
        <width>82</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reboot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEmrAck">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>212</y>
        <width>82</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EMR ACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStartCfgRecovery">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>110</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>199</y>
        <width>166</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelCert">
      <property name="geometry">
       <rect>
        <x>308</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Certificate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetCert">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>225</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GET CERT</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxVariant">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>78</y>
        <width>106</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Select Variant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CDAC</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnFlashCert">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>223</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Flash CERT</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textBrowserLog</tabstop>
  <tabstop>comboBoxComPort</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>comboBoxProject</tabstop>
  <tabstop>btnFwDownload</tabstop>
  <tabstop>btnCertDownload</tabstop>
  <tabstop>btnWrite</tabstop>
  <tabstop>btnCfgRecover</tabstop>
  <tabstop>comboBoxVariant</tabstop>
  <tabstop>btnGetVariant</tabstop>
  <tabstop>btnStartCfgRecovery</tabstop>
  <tabstop>btnGetnSetNwsw</tabstop>
  <tabstop>lineEditNwsw</tabstop>
  <tabstop>btnGetnSetSimtype</tabstop>
  <tabstop>lineEditSimtype</tabstop>
  <tabstop>btnGetCip2</tabstop>
  <tabstop>lineEditCip2</tabstop>
  <tabstop>btnGetUin</tabstop>
  <tabstop>btnSetUin</tabstop>
  <tabstop>lineEditUin</tabstop>
  <tabstop>btnGetVin</tabstop>
  <tabstop>btnSetVin</tabstop>
  <tabstop>lineEditVin</tabstop>
  <tabstop>btnGetCert</tabstop>
  <tabstop>lineEditCert</tabstop>
  <tabstop>btnReboot</tabstop>
  <tabstop>btnEmrAck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
